README.md

```diff
@@ -0,0 +1,90 @@
+<!-- SHIELDS -->
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![License][license-shield]][license-url]
+
+<p>
+  <a href="https://github.com/helsingborg-stad/styleguide">
+    <img src="docs/images/hbg-github-logo-combo.png" alt="Logo" width="300">
+  </a>
+</p>
+<h1>Styleguide</h1>
+<p>
+  <br />
+  <a href="https://github.com/helsingborg-stad/styleguide/issues">Report Bug</a>
+  ·
+  <a href="https://github.com/helsingborg-stad/styleguide/issues">Request Feature</a>
+</p>
+
+
+## Summary
+The style guide is intended for websites within Helsingborgs stad and others who use our platform. The guide provides examples, markup and themes for our standardized components. The Helsingborg Styleguide is a flexible and minimalistic component-based framework built in the BEM standard & designed around the Atomic Design principle.
+
+### Requirements
+
+- NodeJS >= 16
+- PHP >= 7.4
+- [Composer](https://getcomposer.org/)
+
+## Installation
```

Comment:
@sebastianthulin could you please add information to "Installation" to make it complete?

Reply:
Added a little fix for the script installing the component library.
Installation instructions is comlete imo.